Assets/SEE/Controls/SettingsMenue.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+namespace SEE.Controls
+{
+    /// <summary>
+    /// Handles the actions in the
+    /// settings canvas.
+    /// </summary>
+    public class SettingsMenue : MonoBehaviour
```

Comment:
This should be integrated into our existing UI framework within `SEE.UI`, i.e., it should be a `PlatformDependentComponent`. This way, we only have a single canvas for UI components and also have an easier method of adding VR support later on.

As a general outline:
* Make `SettingsMenu` inherit from `PlatformDependentComponent`.
* Then, use the `StartDesktop` and `UpdateDesktop` methods instead of `Start` and `Update`. These methods are only executed on the Desktop.
* Instead of modifying the SEEWorld scene, please do the following:
   * Create a prefab (in `Assets/Resources/Prefabs/UI`) containing your UI component. It should use components from the Modern UI Pack, if possible.
   * In the `SettingsMenu` component, instantiate the prefab and configure any necessary parameters (e.g., the text containing the keybindings). Look at `SimpleMenuDesktop.cs:StartDesktop()` for an example of what this might look like. In general, you should do something like `PrefabInstantiator.InstantiatePrefab(PathToYourPrefabHere, Canvas.transform, false)`. This method then returns the game object that was just created, which you can use to extract the TextMeshPro component.
   * Then, instead of modifying SEEWorld, add this component to the `DesktopPlayer` prefab.

Reply:
I changed it. Please take another look at it, to make sure I implemented it the right way. Thank you.